traversal/focus_test.go

```diff
@@ -326,18 +330,18 @@ func TestFocusedTransformWithLinks(t *testing.T) {
 		n, err := traversal.Progress{
 			Cfg: &cfg,
 		}.FocusedTransform(rootNode, datamodel.ParsePath("linkedMap/nested/nonlink"), func(progress traversal.Progress, prev datamodel.Node) (datamodel.Node, error) {
-			Wish(t, progress.Path.String(), ShouldEqual, "linkedMap/nested/nonlink")
-			Wish(t, must.String(prev), ShouldEqual, "zoo")
-			Wish(t, progress.LastBlock.Path.String(), ShouldEqual, "linkedMap")
-			Wish(t, progress.LastBlock.Link.String(), ShouldEqual, "baguqeeyezhlahvq")
+			qt.Check(t, progress.Path.String(), qt.Equals, "linkedMap/nested/nonlink")
+			qt.Check(t, must.String(prev), qt.Equals, "zoo")
+			qt.Check(t, progress.LastBlock.Path.String(), qt.Equals, "linkedMap")
+			qt.Check(t, progress.LastBlock.Link.String(), qt.Equals, "baguqeeyezhlahvq")
 			nb := prev.Prototype().NewBuilder()
 			nb.AssignString("new string!")
 			return nb.Build(), nil
 		}, false)
-		Wish(t, err, ShouldEqual, nil)
-		Wish(t, n.Kind(), ShouldEqual, datamodel.Kind_Map)
+		qt.Check(t, err, qt.IsNil)
+		qt.Check(t, n.Kind(), qt.Equals, datamodel.Kind_Map)
 		// there should be a new object in our new storage!
-		Wish(t, len(store2.Bag), ShouldEqual, 1)
+		qt.Check(t, len(store2.Bag), qt.Equals, 1)
```

Comment:
nit: qt also has qt.HasLen, which is shorter, and also allows giving better errors (e.g. include the overall value) when the check fails. probably worth checking if this occurs elsewhere across the repo as well.